src/cpp/src/visual_language/pipeline.cpp

```diff
@@ -70,7 +68,7 @@ class ov::genai::VLMPipeline::VLMPipelineImpl {
     // Component for applying sampling to lm outputs
     Sampler m_sampler;
```

Comment:
I suppose base class does not require most of the fields of this class.
I suppose we need to add interface of base class and keep this class as derived one like `VLMPipelineStateful`